### DISCUSSION TITLE
reason why UNIQUE constraint is not set

### DISCUSSION THREAD

**Discussion by rioriost** [apache/age#2135] [Q&A] [answered]
IDs on ._ag_label_vertex / ._ag_label_edge don't have UNIQUE constraint.
If nodes / edges are added via Cypher, it's not a problem.
But when I ingest nodes / edges via PL/PGSQL, I can create them having same ID.

If this specification was defined based on the existing discussion, please ignore my question.

**Comment by MuhammadTahaNaveed** [comment, marked as answer]
@rioriost A PR has already been submitted to address this issue. https://github.com/apache/age/pull/2117

**Reply by rioriost** [reply]
Thank you for letting me know. I'll close the discussion.
